src/components/builder/registration/registration-attribute.spec.tsx

```diff
@@ -38,8 +38,10 @@ test('Available registration attributes are retrieved via context', async () =>
 
   // open the dropdown
   const input = await screen.findByLabelText('Registration attribute');
-  input.focus();
-  await user.keyboard('[ArrowDown]');
+  await act(async () => {
+    input.focus();
+    await user.keyboard('[ArrowDown]');
+  });
```

Comment:
I'm confused: is this `act` call required to suppress the warning? If so, `act` is being called to times, as testing-library apparently handle it already?

Reply:
I think the `input.focus()` also triggers an internal formik state update, perhaps only that needs to be wrapped in `act` :thinking: 

I'll see if I can move the `user.keyboard` out of the `act` when moving tests to jest.